code/src/components/ThoughtInput.css

```diff
@@ -0,0 +1,84 @@
+form {
+  display: flex;
+  flex-direction: column;
+  background: whitesmoke;
+  width: 85%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 20px;
+  padding: 15px;
+  border: darkgray 2px solid;
+  border-radius: 2px;
+  box-shadow: black 8px 8px;
+}
+
+h1 {
+  font-family: 'Roboto', sans-serif;
+  font-size: 16px;
+}
+
+textarea {
+  width: 96%;
+  border: 2px solid lightgray;
+  border-radius: 2px;
+  font-family: 'Inconsolata', monospace;
+  font-size: 16px;
+  outline: none;
+  resize: none;
+  overflow: auto;
+  margin-bottom: 10px;
+  padding-left: 10px;
+  padding-top: 10px;
+}
+
+.submit-button-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.submit-button {
+  border-radius: 25px;
+  background: lightpink;
+  border:none;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  width: 60%;
+  height: 40px;
+  font-family: 'Roboto', sans-serif;
+  font-weight: bolder;
+  outline: none;
+  transition: 0.4s;
+}
+
+.submit-button:hover {
+  box-shadow: 0px 0px 5px 5px #e67687;
+}
+
+span {
```

Comment:
You have done such great job naming your classes, could this one also be specified some more for consistency, sou you easy could see what it refers to?